app/src/renderer/scripts/boot.js

```diff
@@ -117,7 +126,5 @@ export const main = async (
   start = startApp
 ) => {
   const params = getURLParams(window)
-  const enrichedConfig = Object.assign({}, _config, params)
-
-  await start(enrichedConfig)
+  await start(params, _config)
```

Comment:
what is `_config`?

Reply:
the config file import. the name has the lodash to prevent naming conflicts.